ruff.toml

```diff
@@ -1,6 +1,22 @@
 line-length=88
 exclude = [".git","__pycache__"]
+target-version = "py310"
 
 [lint]
-ignore= ["E741"]
-select= ['E', 'F']
\ No newline at end of file
+ignore= ["E741", "RUF005"]
```

Comment:
Why are we ignoring `E741`?

Reply:
It was ignored back when we were using black so I kept it when we transitioned to RUFF.  I can try turning it off if you think it's unnecessary